app/models/osbb.rb

```diff
@@ -0,0 +1,14 @@
+class Osbb < ApplicationRecord
+  before_save :downcase_email
+  
+  validates :name, presence: true, length: { minimum: 3, maximum: 255 }
+  validates :phone, :postcode,  numericality: { only_integer: true }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, length: { maximum: 255 },
+  format: { with: VALID_EMAIL_REGEX }
+  
+  private
+  def downcase_email
```

Comment:
should this method be private? also after 'private' keyword you need to add an empty line. do you have rubocop?

Reply:
i cach why it is private, it is OK, but still add empty line above

Reply:
Resolve